orion/sites/admin/src/announce/announce.styl

```diff
@@ -5,9 +5,48 @@
   height: 400px
   padding: 0 spacing2x
   box-sizing: border-box
-  color: colorBlackDark
+  color: colorGrayDark
+  font-family: fontFamilyRoboto /*not working*/
 
   & h1
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+  & ul
+    padding: 0
+
+  & .announce-lists
+    display: inline-block
+    list-style: none
+    padding-left: 0
+    height: lineHeight6
+    line-height: lineHeight6
+    border-bottom: 1px solid colorGrayLight
```

Comment:
Do you happen to remember CSS property ordering for good style?

I think the order should be
 - display:
 - box-model stuff (width, height, margin, padding, border stuff)
 - text styles (fonts, color, background-color)
 - anything else
 - animation-related lines.